process/release_process.md

```diff
@@ -111,6 +111,31 @@ It leads us to need a separate 'xt-core' repository that will be the entry point
     - Merging the 'development' branch into the 'main' branch
     - Adding the commit that locks all the dependencies
     - Assigning the release tag to the HEAD commit of the 'main' branch
+- **Approach #3**:
+   - Development is done in the 'development' branch. In this branch, all dependencies can stick to the HEAD. In this stage we separate repositories that are including external dependencies and requires actual commit to stabilize external revision. They could be:
+     - Moulin meta repository which includes moulin.yaml
+     - Zephyr based repository with west.yaml which is the dependency for Moulin repository
+   - Release is done by creating a separate branch for each release on top-level repositories with the name of current release
+   - Version format as **<component>-vX.X.X** is acceptable because
+   - Steps to create release are the following:
+     - On repo with external dependensies (moulin or west.yaml) create a separate branch with the release name
+     - Make a stabilizaition commit with the following changes: all external components switched from branch to specific revision.
+     - Repeat this steps for all repo with external dependencies.
+   - Procs:
+     - Release process results the following **git log** for repo with external dependencies:
+       ```
+               v0.1.0              v0.1.1
```

Comment:
Minor comment from my side.

According to the semversion documentation:

https://semver.org/#how-do-i-know-when-to-release-100

```
How do I know when to release 1.0.0?
If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backward compatibility, you should probably already be 1.0.0.
```

**=> I suggest mentioning v1.x.y in the documentation, as many of the developed components are old enough to be outside of the 'initial development phase'.**